client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-form/components/amendment-create-wizard/amendment-create-wizard.component.scss

```diff
@@ -25,6 +25,10 @@
         }
         .paragraph-text {
             flex: 1;
+
+            .insert {
+                text-decoration: underline !important;
+            }
```

Comment:
Mmh, I haven't found the usage of this. Is this debug code?

Reply:
This fixes the missing underline if the previous amendment created a completely new paragraph
So no debug code :D